lib/client.js

```diff
@@ -138,12 +81,21 @@ class Client {
       res.setHeader('Keep-Alive', 'timeout=' + timeout);
     }
     if (this.ip) {
-      this.ip = ipv6toipv4(this.ip);
-      this.ipInt = api.common.ipToInt(this.ip);
+      const ip = api.net.isIPv6(this.ip) ? this.ip.slice(7) : this.ip;
```

Comment:
I am not sure this will work with any IPv6 address, for example, if we get `2001:db8::ff00:42:8329`, the result of executing this line will set the `ip` value to `8::ff00:42:8329` which is an entirely different IPv6 address.
If this was supposed to be a workaround for [IPv4-Mapped IPv6 Addresses](https://tools.ietf.org/html/rfc4291#section-2.5.5.2), then we should check if it is the case for each IP address and only then extract the IPv4 out of it if needed.

Reply:
We have separate issue for this: https://github.com/metarhia/impress/issues/889